bitsv/wallet.py

```diff
@@ -148,7 +154,7 @@ def __init__(self, wif=None):
         self.unspents = []
         self.transactions = []
         self.network = 'main'
```

Comment:
You may wonder why I have still kept the self.network = 'main' (not used)... it is because it may become used in next PR...